stripe/src/main/java/com/stripe/android/StripeNetworkUtils.java

```diff
@@ -136,14 +135,17 @@ public static void removeNullAndEmptyParams(@NonNull Map<String, Object> mapToEd
 
             if (mapToEdit.get(key) instanceof CharSequence) {
                 CharSequence sequence = (CharSequence) mapToEdit.get(key);
-                if (TextUtils.isEmpty(sequence)) {
+                if (StripeTextUtils.isEmpty(sequence)) {
                     mapToEdit.remove(key);
                 }
             }
 
             if (mapToEdit.get(key) instanceof Map) {
-                Map<String, Object> stringObjectMap = (Map<String, Object>) mapToEdit.get(key);
-                removeNullAndEmptyParams(stringObjectMap);
+                final Map<String, Object> stringObjectMap =
+                        (Map<String, Object>) mapToEdit.get(key);
+                if (stringObjectMap != null) {
```

Comment:
I don't think this null check is necessary since we know mapToEdit.get(key) is an instance of Map (not null).

Reply:
I'm going to clean up this method in a follow-up PR